LongWarOfTheChosen/Localization/LW_PerkPack_Integrated/XComGame.int

```diff
@@ -279,9 +281,9 @@ LocPromotionPopupText="<Bullet/> Once per mission, fire a pistol shot that canno
 [Gunslinger X2AbilityTemplate]
 LocFriendlyName="Gunslinger"
 LocFlyOverText="Gunslinger"
-LocLongDescription="Take a reaction shot with your pistol against any enemy that moves or attacks within 12 tiles and a wide cone of fire."
-LocHelpText="Enhanced pistol overwatch in a cone of fire."
-LocPromotionPopupText="<Bullet/> Range is 12 tiles.<br/><Bullet/> Three-turn cooldown.<br/>"
+LocLongDescription="Take a reaction shot that can crit with your pistol against any enemy that moves or attacks within 8 tiles."
```

Comment:
Okay, but the parameter being displayed in meters will be confusing.

Reply:
You can convert to tiles in the tag implementation.

Reply:
okay, that's what i did.

Reply:
OK, I see the source of the confusion. There's still a literal '8' in the above text when it can use the existing `<Ability:GUNSLINGER_TILES_RANGE/>` tag. I'll fix that.